### PR TITLE
Oleks branch

### PR DESCRIPTION
**Pull request by oleks2oleks**
Hey, Demi, could you please review my code for passenger class + Domain

### CODE REVIEW

File: src/main/java/controller/PassengersController.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+                System.out.println(myPassengerId
+                        + " " + myPassengerFullName
+                        + " " + myPassengerEmail
+                        + " " + myPassengerPhoneNumber
+                        + " " + myPassengerAge
+                        + " " + myPassengerStartLocationId
+                        + " " + myPassengerStopLocationId
+                        + " " + myPassengerBusID
+                        + " " + timestamp);
```
Comment (demiavaliani) [inline src/main/java/controller/PassengersController.java]: Maybe just to make the output more beautiful and readable add dash between the values. 
`myPassengerId + " - " + myPassengerFullName`

File: src/main/java/controller/PassengersController.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                String myPassengerFullName = resultSetPassengers.getString("fullName");
+                String myPassengerEmail = resultSetPassengers.getString("email");
+                String myPassengerPhoneNumber = resultSetPassengers.getString("phoneNumber");
+                int myPassengerAge = resultSetPassengers.getInt("age");
+                int myPassengerStartLocationId = resultSetPassengers.getInt("startLocationId");
+                int myPassengerStopLocationId = resultSetPassengers.getInt("stopLocationId");
+                int myPassengerBusID = resultSetPassengers.getInt("busId");
+                Timestamp timestamp = resultSetPassengers.getTimestamp("createdOn");
+
+                if (myPassengerId == id) {
```
Comment (demiavaliani) [inline src/main/java/controller/PassengersController.java]: This `if (myPassengerId == id)` statement does not work here. It does not check anything. Instead change `while` to `if` in the beginning.

File: src/main/java/controller/PassengersController.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+     and search mysql database using jdbc for the bus and show the results.)
+     */
+    public void findByPassengerID(int id) {
+        Statement statement = databaseHandler.createStatement();
+        try {
+            ResultSet resultSetPassengers = statement.executeQuery("SELECT * FROM passengers WHERE id = " + id +";");
+            System.out.println("=========================================");
+            System.out.println("Passenger by id # " + id + "information from 'passengers' table");
+
+            while (resultSetPassengers.next()) {
```
Comment (demiavaliani) [inline src/main/java/controller/PassengersController.java]: Change `while` to `if`

File: src/main/java/controller/PassengersController.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+                            + " " + myPassengerFullName
+                            + " " + myPassengerEmail
+                            + " " + myPassengerPhoneNumber
+                            + " " + myPassengerAge
+                            + " " + myPassengerStartLocationId
+                            + " " + myPassengerStopLocationId
+                            + " " + myPassengerBusID
+                            + " " + timestamp);
+                }
+            }
```
Comment (demiavaliani) [inline src/main/java/controller/PassengersController.java]: `else {
Such ID does not exist!}`

File: src/main/java/domain/Domain.java [hunk 5]
```diff
@@ -2,18 +2,44 @@
...
+								2,
+								3);
+		passengersController.listAllPassengers();
+
+		System.out.println("=========================================");
+		System.out.println("Passenger with id #3 information:");
+		passengersController.findByPassengerID(3);
+		/*
+		@Oleks
+		 */
```
Comment (demiavaliani) [inline src/main/java/domain/Domain.java:43]: Please don't add extra :D

**Review by demiavaliani — CHANGES_REQUESTED** [review]
You should definitely adjust the if() statement, in case the input does not exist in the table. There should be some sort of message in such case.

File: src/main/java/controller/PassengersController.java (hunk 1 above)
Comment (oleks2oleks) [reply src/main/java/controller/PassengersController.java]: done

File: src/main/java/controller/PassengersController.java (hunk 2 above)
Comment (oleks2oleks) [reply src/main/java/controller/PassengersController.java]: done

File: src/main/java/controller/PassengersController.java (hunk 3 above)
Comment (oleks2oleks) [reply src/main/java/controller/PassengersController.java]: done

File: src/main/java/controller/PassengersController.java (hunk 4 above)
Comment (oleks2oleks) [reply src/main/java/controller/PassengersController.java]: done

File: src/main/java/domain/Domain.java (hunk 5 above)
Comment (oleks2oleks) [reply src/main/java/domain/Domain.java:43]: too late

**Pull request closed without merging**